aten/src/ATen/native/native_functions.yaml

```diff
@@ -5031,6 +5031,7 @@
 
 - func: relu6(Tensor self) -> Tensor
   python_module: nn
+  tags: pointwise
 
 - func: relu6_(Tensor(a!) self) -> Tensor(a!)
```

Comment:
The inplace versions should also be marked as pointwise, right?

Reply:
Ah, but lots of other inplace ops are already tagged as pointwise unfortunately.

Reply:
hm true, I think I'll open a follow-up to handle this properly

Reply:
oh, this actually already works today:
```python
def test_inplace_unary_op_returns_same_instance(self, device, dtype):
    nt = random_nt_from_dims(
        [3, None, 5],
        device=device,
        dtype=dtype,
        layout=torch.jagged,
    )
    out = nt.relu_()
    # passes!
    self.assertIs(out, nt)
```

What I believe happens is that the ADInplaceOrView key handles aliasing.